lib/utils/url_launcher.dart

```diff
@@ -1,9 +1,12 @@
+import 'package:url_launcher/url_launcher.dart';
 import 'package:url_launcher/url_launcher_string.dart';
 
 void launchURL(String url) async {
-  if (await canLaunchUrlString(url)) {
-    await launchUrlString(url);
+  var str1 = 'mailto';
+  if (url.startsWith(str1)) {
```

Comment:
You can directly write the string inside startswith(...) as str1 is not used anywhere else

Reply:
used the string directly and removed str1.